integration_tests/src/main/python/time_zone_utils.py

```diff
@@ -0,0 +1,630 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# get from Java:
+# ZoneId.getAvailableZoneIds
+# id.getRules.isFixedOffset || id.getRules.getTransitionRules.isEmpty
```

Comment:
Hmm, can we use Python Java gateway (e.g., ```Py4j```) to obtain the list instead of hardcoding it here? Besides the maintenance, just want to ensure we're consistent with Java on this behavior ```id.getRules.isFixedOffset || id.getRules.getTransitionRules.isEmpty```.

Reply:
It's invoked in the pytest marks, Python Java gateway is not  available when Spark session is not started.

Reply:
Done. Now is using the following:
```
@pytest.mark.skipif(is_supported_time_zone() ......)
```

The is_supported_time_zone works in the pytest marks.